### PR TITLE
Absolute clamp

### PR DESCRIPTION
**Pull request by silviacasac**
First pull request for the **Absolute clamp** transformation, which corresponds to [this GitHub permalink](https://github.com/opendp/opendp/blob/main/rust/opendp/src/trans/clamp.rs#L46-L72). This is the same as the clamp transformation but applied to elements instead of vectors, and so using absolute distance instead of symmetric distance.

Files:
- abs_clamp.tex -- the LaTeX document with the proof.
- abs_clamp.pdf -- the PDF generated from the TeX document.

Flags: once the row-by-row transform lemmas are pushed, there should be a better way of presenting symmetric distance clamp / absolute distance clamp / row by row transform approach. The new OpenDP code also uses a clampable domain now (which avoids repetitions between symmetric distance clamp and absolute clamp), but for now we wanted to keep the proofs separate. This is why the clampable domain is not mentioned in the proof, but there should also be a better way of presenting the 2 versions together.

Edit: this is a more general issue, e.g., Laplace also has a `VectorDomain` version and an `AllDomain` version.

### CODE REVIEW

File: abs_clamp/abs_clamp.tex [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    
+\silvia{Flag: this is an example of the subdomain issues that we have been discussing during the week of July 19. Hence this paragraph might need some phrasing updates when the compatibility pairing constructor and the subdomain trait are implemented.}
+
+\smallskip
+\textbf{(Stability guarantee).} Throughout the stability guarantee proof, we can assume that $\function(x)$ and $\function(y)$ are in the correct output domain, by the \textit{appropriate output domain property} shown above.
```
Comment (Shoeboxam) [inline abs_clamp/abs_clamp.tex:187]: I don't see how this shows that right stability relation. I think your cases are as follows:
1. If the clamping range is less than d_in
2. If the clamping range is greater than d_in

The relation you've shown `d_in <= d_out` doesn't tighten the sensitivity when the clamping range is less than the input distance.

Comment (silviacasac) [reply abs_clamp/abs_clamp.tex:187]: Sorry, I didn't tighten the third case. It is fixed now.

**Comment by silviacasac** [comment]
Remaining things to check: none I think, except whether we want to actually have a constructor for scalar clamp with absolute value or if we should delete it (given that it does not appear to have much use).

**Comment by Shoeboxam** [comment]
We decided against keeping this constructor in the library because it didn't have sufficient utility. Sorry for pushing you to make this proof! Closing this PR to prevent unnecessary vetting work. It remains in the closed PR's section. Feel free to reopen if you don't agree.

**Pull request closed without merging**